tools/zbrealign

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
```

Comment:
Have you had success with realignment frames on commercial devices? We haven't had success with these in-the-wild in the past. Would be interested to learn more - perhaps discuss offline.